apps/aecore/src/aec_blocks.erl

```diff
@@ -70,3 +83,23 @@ to_header(#block{height = Height,
             nonce = Nonce,
             time = Time,
             version = Version}.
+
+-spec serialize_for_network(BlockInternalRepresentation) ->
```

Comment:
do we need _for_network suffix? Serialization format is unified for all APIs

Reply:
We may have several serialization formats. In the previous testnet we used to persist a serialized format that used to include not only items serialized in the network but also internal representation items i.e.:
* Pointer to persisted state Merkle tree at that block;
* Hashes of ancestor block headers in addition to the previous one only - so that to enable jumping e.g. 100 blocks back in the chain without needing to do 100 lookups.

Reply:
And cumulative/total difficulty.

Reply:
@lucafavatella Please remember to keep it simple! We don't need an internal serialization format.